src/api/operator/operatorApi.ts

```diff
@@ -150,6 +159,24 @@ export async function getTrades(
   return [...trades[0], ...trades[1]]
 }
 
+/**
+ * Gets information about solver competition By Transaction Hash
+ *  Required params:
+ *   - txHash: string
+ *
+ */
+export async function getSolverCompetitionByTx(params: GetTxSolverCompetitionParams): Promise<RawSolverCompetition> {
+  const { networkId, txHash } = params
+
+  console.log(
+    `[getSolverCompetitionByTx] Fetching Solver Competition on network ${networkId} with filters: tx_hash=${txHash}`,
+  )
+
+  const queryString = '/solver_competition/by_tx_hash/' + txHash
```

Comment:
Just noticed, this endpoint is available from `orderBookSDK` since version https://github.com/cowprotocol/cow-sdk/releases/tag/v2.3.0

Update to the latest and you can use it directly.

You will still HAVE TO query both `barn` and `prod` envs, like it's done here: https://github.com/cowprotocol/explorer/blob/develop/src/api/operator/operatorApi.ts#L104-L119